include/openenclave/attestation/plugin.h

```diff
@@ -293,16 +291,21 @@ oe_result_t oe_unregister_attester(oe_attester_t* plugin);
  */
 oe_result_t oe_unregister_verifier(oe_verifier_t* plugin);
 
+#if (OE_API_VERSION < 3)
+#define oe_get_evidence oe_get_evidence_v2
+#define oe_get_evidence_v2(p1, p2, p3, p4, p5, p6, p7, p8, p9, p10) \
+    oe_get_evidence_v3(p1, p3, p4, p5, p6, p7, p8, p9, p10)
```

Comment:
@CodeMonkeyLeet this mechanism allows for calling `oe_get_evidence()` with the old parameters but it still breaks the semantics (flags will just be ignored). Is this considered ok?

Reply:
The removed flags is used to indicate if the attestation is for local or remote. When the flags parameter is ignored, default is remote.

---
In reply to: [383453867](https://github.com/openenclave/openenclave/pull/2579#discussion_r383453867) [](ancestors = 383453867)

Reply:
This still changes how the function works. If I call oe_get_evidence with the local flag, it will just be ignored and I'll get the remote version. I consider that still broken even the API version define is used.

Reply:
Will ensure OE_EVIDENCE_FLAGS_LOCAL_ATTESTATION is honor when OE_API_VERSION is 2.

Plugin design requires the caller to manually register attester/verifier. This PR does not change the behavior.

---
In reply to: [383582536](https://github.com/openenclave/openenclave/pull/2579#discussion_r383582536) [](ancestors = 383582536)